addons/sourcemod/scripting/MyJailbreak/Modules/Warden/orders.sp

```diff
@@ -258,6 +262,10 @@ void Command_Handler(char[] number)
 		}
 	}
 	else SetFailState("MyJailbreak Warden - Can't open File: %s", g_sMenuFile);
+	
+	// prevent memory leaks
+	delete kvMenu;

```

Comment:
You should either use the syntax style as other stocks (check if the file handle is null and if it is setfailstate) or place delete in the lower block.

Reply:
You can close this pr. I will send another pr when I have more time to see all the possible memory leaks.

Reply:
We could have left this pr, and keep making updates on it, but if you want to do so...